app/Models/CaseStudy.php

```diff
@@ -25,21 +26,24 @@ class CaseStudy extends Model
     public const string PHOTOS_DIRECTORY = "case-studies";
 
     public $translatable = [
-        "name",
-        "description",
+        "name_1",
+        "name_2",
+        "name_3",
```

Comment:
```suggestion
        "first_name",
        "second_name",
        "third_name",
```
should not be better?

Reply:
Changed to sth similar: `name_first part, name_second_part` and `name_third_part`